app/src/css/vendor/bootstrap/overrides/_card.scss

```diff
@@ -0,0 +1,28 @@
+.card-wrapper {
+  display: flex;
+  justify-content: space-between;
```

Comment:
This ticket is focused on the implementation and function. We can look at the flex grid and how wide the columns and padding will be in the style ticket.

Reply:
The style ticket is [1907](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1907).

Reply:
Thanks @jjmccoy! Also, since the number of cards will change once we have the actual bucket names, etc, I'm sure the style implementation will change as well

Reply:
gotcha. thanks